gemini/gradio-voice/app.py

```diff
@@ -0,0 +1,134 @@
+import asyncio
+import base64
+
+from google import genai
+import gradio as gr
+from gradio_webrtc import AsyncStreamHandler, WebRTC, async_aggregate_bytes_to_16bit
+import numpy as np
+
+
+def encode_audio(data: np.ndarray, sample_rate: int) -> str:
```

Comment:
`sample_rate` doesn't seem to be used. Should it be input to one of the function calls?

Reply:
Oversight on my part, it's not needed in the function signature.